src/IntervalLinearAlgebra.jl

```diff
@@ -42,9 +42,23 @@ include("pils/pils_solvers.jl")
 include("eigenvalues/interval_eigenvalues.jl")
 include("eigenvalues/verify_eigs.jl")
 
+using LinearAlgebra
+
+if Sys.ARCH == :x86_64
+    using OpenBLASConsistentFPCSR_jll
+end
+
 function  __init__()
     @require IntervalConstraintProgramming = "138f1668-1576-5ad7-91b9-7425abbf3153" include("linear_systems/oettli_nonlinear.jl")
     @require LazySets = "b4f0291d-fe17-52bc-9479-3d1a343d9043" include("linear_systems/oettli_linear.jl")
+    if Sys.ARCH == :x86_64
+        @info "Switching to OpenBLAS with ConsistentFPCSR = 1 flag enabled, guarantees
+        coherent floating point rounding mode over all threads"
```

Comment:
```suggestion
        correct floating point rounding mode over all threads."
```

(sorry for the nitpick)